ipalib/misc.py

```diff
@@ -26,26 +26,31 @@ class env(LocalOrRemote):
     )
 
     takes_options = LocalOrRemote.takes_options + (
-        Flag('all',
+        Flag(
```

Comment:
If you look at some other newly written plugins (e.g. `ipaserver/plugins/server.py`) you will se that I have used the option formatting constistent with them and I would rather keep it  like that.